rllib/core/rl_module/rl_module.py

```diff
@@ -232,43 +232,62 @@ class RLModule(abc.ABC):
     Subclasses should call super().__init__(config) in their __init__ method.
     Here is the pseudocode for how the forward methods are called:
 
-    During Training (acting in env from each rollout worker):
+    .. testcode::
 
-    .. code-block:: python
+        # Example for creating a sampling loop:
```

Comment:
This example is long. For better readability, it might be better to split this into separate testcodes (one for sampling, training, inference, etc)